R/fit.R

```diff
@@ -330,7 +327,7 @@ CmdStanFit$set("public", name = "lp", value = lp)
 #'
 summary <- function(variables = NULL, ...) {
   draws <- self$draws(variables)
-  if (self$runset$method() == "sample") {
+  if (self$metadata()$method == "sample") {
```

Comment:
We need to check the method via metadata()$method and not runset$method() or it won't work with the objects created after read_cmdstan_csv